osbs/build/build_request.py

```diff
@@ -791,6 +832,9 @@ def render(self, validate=True):
         self.render_koji_promote(use_auth=use_auth)
         self.render_sendmail()
 
+
+        self.render_customizations()
```

Comment:
I think it should be done at the start, i.e. it describes changes that you could make to the prod_inner.json file.

If there are changes that a site needs to make that get overwritten by the way osbs-client transforms the plugins during rendering, we need to handle that in osbs-client either by changing its default behaviour or by making it configurable.